build.gradle.kts

```diff
@@ -143,7 +156,7 @@ bintray {
     user = "oshai"//project.hasProperty("bintrayUser") ? project.property("bintrayUser") : System.getenv("BINTRAY_USER")
     key = "mykey" //https://bintray.com/profile/edit
     // project.hasProperty("bintrayApiKey") ? project.property("bintrayApiKey") : System.getenv("BINTRAY_API_KEY")
-    setPublications("metadata", "jvm", "js")
+    setPublications("metadata", "jvm", "js", "linuxX64")
```

Comment:
Might not immediately work with native targets. https://github.com/bintray/gradle-bintray-plugin/issues/229 .

Reply:
Removed **linuxX64** from the list. There will be an opportunity to revert back after the issue is fixed with the Bintray Gradle plugin.